app/views/pets/show.html.erb

```diff
@@ -5,8 +5,18 @@
 <p>Approximate Age: <%= @pet.approximate_age %></p>
 <p>Sex: <%= @pet.sex %></p>
 <p>Adoption Status: <%= @pet.adoption_status %></p>
-
+<!-- if favorites.include?(pet_id)
+  @favorite_link_text = "Remove Favorite"
+  @favorite_link_method = :delete
+else
+  @favorite_link_text = "Add Favorite"
+  @favorite_link_method = :patch
+end -->
 <ul>
```

Comment:
What is this commented out section? Do we need to delete this or uncomment this out and re-run the test?

Reply:
Good catch. That was from us dream driving together.